duckduckgo/Android
# Hotfix/5.165.1

Task/Issue URL: https://app.asana.com/0/488551667048375/1205216014376915/f

### Description

### Steps to test this PR

- Install from develop and let app run to get remote config
- Install from this branch on top, and then ensure logins is available again
